vueui/README.md

```diff
@@ -185,10 +185,14 @@ Example:
 ```
 Parameters:
  - `$slot` - Contents of button.
- - `params` - key value pairs to send to `Topic` of object.
+ - `params` - key value pairs to send to `Topic` of object. Can contain objects or arrays (DM keyed lists and lists respectively).
+ - `raw-params` - key value pairs to send to `Topic`.
```

Comment:
```suggestion
 - `unsafe-params` - Used to execute a generic `Topic()` call to the game. Requires that you specify the `src` object as a valid reference, otherwise it will not function. **Should not generally be used**, primary use-case is backwards compatibility with older APIs that are spread out over multiple objects.
```

this. Explains a bit more and gives the proper connotation.